EIPS/eip-1985.md

```diff
@@ -50,15 +51,22 @@ They restrict the results (i.e. values pushed to the stack) of the instructions
    - `CREATE` (`0xf0`),
    - `CREATE2` (`0xf5`).
 
-4. *buffer size*
+4. *buffer size*, *code size*, *memory size*
    is a range between `0` and `0xffffffff` (`2**32 - 1`, `4294967295`).
    It affects the following instructions:
    - `CALLDATASIZE` (`0x36`),
    - `CODESIZE` (`0x38`),
    - `EXTCODESIZE` (`0x3b`),
    - `RETURNDATASIZE` (`0x3d`),
-   - `MSIZE` (`0x59`).
+   - `MSIZE` (`0x59`),
+   - `PC` (`0x58`).
+
+They restrict the acceptable input value ranges (as defined above) of the instructions listed below.
+If a value outside of this ranges are encountered, the execution stops with an exceptional halt consuming all remaining gas.
+
+1. CALL* / CREATE*
 
+2. CALLDATACOPY / CODECOPY / EXTCODECOPY / RETURNDATACOPY / CALLDATALOAD / MLOAD / MSTORE / MSTORE8 / REVERT / RETURN
```

Comment:
@chfast any idea how to write this section nicely?

Reply:
@chfast ping

Reply:
Have some ideas, not little time at the moment.

Reply:
Moved this commit to #2276.